pipelines/cpFlat.yaml

```diff
@@ -5,18 +5,10 @@ tasks:
     config:
       connections.ccdExposure: 'raw'
       connections.outputExposure: 'cpFlatProc'
-      doBias: True
-      doVariance: True
-      doLinearize: True
-      doCrosstalk: True
-      doDefect: True
-      doNanMasking: True
-      doInterpolate: True
-      doDark: True
+      doWrite: True
       doBrighterFatter: False
       doFlat: False
       doFringe: False
-      doApplyGains: False
```

Comment:
`doApplyGains` needs to be disabled here, to ensure that the flat constructed doesn't have that gain scaling applied.  `doFlat` and `doApplyGains` can't both be enabled, so a flat constructed with the scaling can't be applied correctly.